site/layouts/_default/examples.html

```diff
@@ -42,16 +42,6 @@
   <body{{ with .Page.Params.body_class }} class="{{ . }}"{{ end }}>
     {{ .Content }}
 
-    {{ if ne .Page.Params.include_js false }}
-      {{ if eq (getenv "HUGO_ENV") "production" }}
-        <script src="/docs/{{ .Site.Params.docs_version }}/dist/js/bootstrap.bundle.min.js" integrity="{{ .Site.Params.cdn.js_bundle_hash }}" crossorigin="anonymous"></script>
-      {{ else }}
-        <script src="/docs/{{ .Site.Params.docs_version }}/dist/js/bootstrap.bundle.js"></script>
-      {{ end }}
-
-      {{ range .Page.Params.extra_js }}
-        <script src="{{ . }}"></script>
-      {{- end }}
-    {{ end }}
+    {{ partial "scripts" . }}
```

Comment:
We should probably revert any layout changes and use a separate layout for the new example like we do with everything else.

Reply:
I did this so that it would be possible for all layouts to add `extra_js`. Are you suggesting to duplicate the `single` layout specifically for examples with a docs-look?

Reply:
The Masonry example should use the examples layout like the other examples I think.